src/components/primary-cta.jsx

```diff
@@ -2,7 +2,30 @@ import React from 'react';
 import CtaRocketship from '../images/svg-2022/cta-rocketship';
 
 const PrimaryCta = ({ link, textDescription }) => {
-    return <a
+    if (link == null) {
+        const clicked = async e => {
+            e.preventDefault();
+            alert("Copied the HBP website to clipboard, please share with your friends!");
```

Comment:
instead of doing `alert` (it's getting removed from browsers soon i think) can you instead do either a small bubble (that doesn't interrupt user experience, ie block scrolling or require dismissal) or change the text displayed in the CTA to "link copied to clipboard" for a few seconds?

Reply:
yeah I was considering that, similar to the small tab on the registration portal that pops up when an applicant has submitted their application
do you know which file that's in?

Reply:
sorry for late reply–so you won't be able to do the same thing as the application portal because that uses ant design, and ant design isn't installed for the website (and it idk if we should add it just for the one notification component).

it's [right here](https://github.com/HackBeanpot/application-portal/blob/main/pages/application.tsx#L112) though, if you want to take a look anyway.

what you can do instead is change the text for a few seconds on click, so to do that probably the easiest way would be to do something like this:


```javascript
const [isClicked, setIsClicked] = useState(false);
const ctaText = isClicked ? "Copied to clipboard!" : <the current message>
const onClick = () => {
    setIsClicked(true);
    // second argument is in milliseconds, so 3 * 1000 = 3 seconds
    // after 3 seconds, setTimeout will call the function you provide
    setTimeout(() => setIsClicked(false), 3 * 1000);
}

// pass the Cta component the text as the textDescription prop, see:
// https://github.com/HackBeanpot/website-2022/blob/develop/src/components/primary-cta.jsx
return <button onClick={onClick}><CtaRocketship textDescription={ctaText} /></button>;
```